src/getMachines.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+from decEncoder import *
+from boto3.dynamodb.conditions import Key, Attr
+
+"""Scan dynamo table"""
+dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')
+machineTable = dynamodb.Table('Machines')
+
+"""Lambda handler function for /buildings/{buildingId}/machines API call
+   Returns all the machines in a given building
+   or machines matching query parameters (machineType and/or status)
+   or 'Building or machine not found' error if no machines are returned matching criteria
+   or 'No path parameters' if there are no path parameters
+"""
+def getMachines(event, context):
+    buildingId = None
```

Comment:
This variable definition isn't really needed, as it isn't used outside of the if statement.